app/controllers/event.js

```diff
@@ -0,0 +1,121 @@
+const Event = require('../models/Event')
+module.exports = {
+  createEvent: async (req, res, next) => {
+    const event = new Event(req.body)
+    try {
+      await event.save()
+      res.status(201).json({ event: event })
+    } catch (error) {
+      res.status(400).json({ error: error.errors })
+    }
+  },
+  updateEvent: async (req, res) => {
+    Event.findById(req.params.id, (err) => {
+      if (err) {
+        res.status(400).json({ error: 'Did not find an event!' })
+      } else {
+        Event.findByIdAndUpdate(req.params.id, req.body, function (err, updatedEvent) {
+          if (err) {
+            res.status(400).json({ error: 'Cannot Update the data' })
+          } else {
+            res.status(201).json({ updatedEvent: updatedEvent })
+          }
+        })
+      }
+    })
+  },
+  rsvp: async (req, res) => {
+    const { yes, no, maybe } = req.body
+    Event.findById(req.params.id, function (error, data) {
```

Comment:
try to change to callbacks to await statement. In other files we are following await and this paradigm should not break.

Reply:
i missed sorry about that will update soon